mir_eval/util.py

```diff
@@ -616,9 +664,8 @@ def match_events(ref, est, window):
         ``matching[i] == (i, j)`` where ``ref[i]`` matches ``est[j]``.
 
     """
-
     # Compute the indices of feasible pairings
-    hits = np.where(np.abs(np.subtract.outer(ref, est)) <= window)
+    hits = np.where(outer_distance(ref, est) <= window)
```

Comment:
I thought about that - this abstracts the distance computation, but only to the minimal extent needed for my application ;). I figured it was best to keep it as simple as possible for now, and if someone has a use case in the future that needs more abstraction we can change it. Until then, I think it's overkill to abstract more. Just my 2 cents.